sql/1.107/github_commit_comments_table.sql

```diff
@@ -0,0 +1,18 @@
+BEGIN;
+
+CREATE TABLE "github_commit_comment" (
+    "id" serial NOT NULL,
```

Comment:
Right, I get that.   As with `contributor_activity` it's just extra data since it's not being used to, say, make a primary key.  Not a big deal just wondering.